mammoth/inputters/dataset.py

```diff
@@ -162,13 +184,13 @@ def _cast(example_dict):
                 if self.transforms is not None else lambda x: x
             ),
             stride=self.stride,
-            offset=self.offset,
+            offset=offset,
         )
         examples = map(_cast, examples)
         yield from examples
 
     # FIXME: some RNN archs require sorting src's by length
```

Comment:
Out of this PR concenrs... do we still support RNNs? if not, we can just delete this

Reply:
no, we no longer support RNNs. the move is to externalize transformer variantsvia lucidrains (see #56). Will remove the comment